ai-engineering/concepts.mdx

```diff
@@ -55,18 +60,26 @@ Ground truth is the validated, expert-approved correct output for a given input.
 
 Annotations are expert-provided labels, corrections, or outputs added to records to establish or refine ground truth.
 
-### Grader
+### Scorer
+
+A scorer is a function that evaluates a capability’s output. It programmatically assesses quality by comparing the generated output against ground truth or other criteria, returning a score or judgment. Scorers are the reusable, atomic scoring logic used in all forms of evaluation.
+
+### Flag
+
+A flag is a configuration parameter that controls how your AI capability behaves. Flags let you parameterize aspects like model choice, temperature, prompting strategies, or retrieval approaches. By defining flags, you can run experiments to compare different configurations and systematically determine which approach performs best.
+
+### Experiment
 
-A grader is a function that scores a capability’s output. It programmatically assesses quality by comparing the generated output against ground truth or other criteria, returning a score or judgment. Graders are the reusable, atomic scoring logic used in all forms of evaluation.
+An experiment is an evaluation run with a specific set of flag values. By running multiple experiments with different flag configurations, you can compare performance across different models, prompts, or strategies to find the optimal setup for your capability.
 
-### Evaluator (eval)
+### Evaluator or "eval"
 
-An evaluator, or eval, is the process of testing a capability against a collection of ground truth data using one or more graders. An eval runs the capability on every record in the collection and reports metrics like accuracy, pass-rate, and cost. Evals are typically run before deployment to benchmark performance.
+An evaluator, or eval, is the process of testing a capability against a collection of ground truth data using one or more scorers. An eval runs the capability on every record in the collection and reports metrics like accuracy, pass-rate, and cost. Evals are typically run before deployment to benchmark performance.
 
 ### Online eval
 
-An online eval is the process of applying a grader to a capability’s live production traffic. This provides real-time feedback on performance degradation, cost, and quality drift, enabling continuous monitoring and improvement.
+An online eval is the process of applying a scorer to a capability’s live production traffic. This provides real-time feedback on performance degradation, cost, and quality drift, enabling continuous monitoring and improvement.
 
 ## What’s next?
 
```

Comment:
(somehow cant find it in the diff, but noticed this in the deployment:

"Reference
A reference is a historical example of a task completed successfully, serving as a benchmark for AI performance. References provide the input-output pairs that demonstrate the expected behavior and quality standards."

never heard us use that term internally before